ThirdPartyHosting.md

```diff
@@ -71,6 +71,13 @@ Here are some good resources where you learn more about the importance of full n
 * [Why Your Business Should Use a Full Node to Accept Bitcoin](https://en.bitcoin.it/wiki/Why_Your_Business_Should_Use_a_Full_Node_to_Accept_Bitcoin)
 * [Clearing Up Misconceptions About Full Nodes](https://en.bitcoin.it/wiki/Clearing_Up_Misconceptions_About_Full_Nodes)
 
+### Trust Concerns 
+Third Party hosts have the ability to enable specific features for their non-admin users which require users to place some level of trust in the Third Party host, if such features are used. 
+
+Specifically, Third Party hosts should not enable the policies "Allow non-admins to create hot wallets for their stores" or "Allow non-admins to import their hot wallets to the node wallet" without understanding that users will be using hot wallets on the server. This feature is disabled by default for non-admins to reduce risk considerations for both Third Party hosts and their users.
```

Comment:
There is also `Allow non-admins to use the internal lightning node in their stores` which is dangerous